target/ppc/insn32.decode

```diff
@@ -128,6 +128,9 @@
 %xx2_xb         1:1 11:5
 @XX2            ...... ..... ... uim:2 ..... ......... ..       &XX2 xt=%xx2_xt xb=%xx2_xb
 
+&X_bf_xb        bf xb
+@X_bf_xb        ...... bf:3 -- ..... ..... ......... . -        &X_bf_xb xb=%xx2_xb
```

Comment:
Get rid of the '--' here, and replace it with '..'.

It's better to keep the '-'s only on the instruction patterns, not on the formats

Reply:
done, thanks